lib/questions.js

```diff
@@ -66,6 +68,19 @@ export const jsQuestions = [
             { message: "Browser", name: "browser" },
             { message: "Node", name: "node" }
         ]
+    },
+    {
+        type: "select",
+        name: "configFileLanguage",
+        message: "What language do you want your configuration file?",
```

Comment:
[Grammar] This is technically grammatically incorrect. I'd go with something like:

```suggestion
        message: "Which language do you want your configuration file be written in?",
```

...or something like that.